android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     dependencies {
```

Comment:
In my setup I cannot build without updated kotlin version:
ext.kotlin_version = '1.5.31'